x-pack/plugins/ingest_manager/common/types/models/datasource.ts

```diff
@@ -5,31 +5,23 @@
  */
 
 export interface NewDatasource {
-  name: string;
-  namespace?: string;
-  config_id: string;
-  enabled: boolean;
+  enabled?: boolean;
+  title?: string;
   package?: {
-    assets: Array<{
-      id: string;
-      type: string;
-    }>;
-    description: string;
     name: string;
-    title: string;
     version: string;
   };
-  output_id: string;
+  namespace?: string;
+  use_output: string;
```

Comment:
This is an example where it's ok that the field name differs in SO than agent schema. The field names will be remapped, but the SO name is more friendly to work with in UI.